journal/journal_unix.go

```diff
@@ -65,10 +65,14 @@ func Enabled() bool {
 		return false
 	}
 
-	if _, err := net.Dial("unixgram", journalSocket); err != nil {
+	conn, err := net.Dial("unixgram", journalSocket)
+
+	if err != nil {
 		return false
 	}
 
+	defer conn.Close()
```

Comment:
Let move this one line up, no need to have an extra empty line after the if-err body.

Reply:
I've squashed the two commits into one and removed spurious blank lines.  Does this need a unit test?

Reply:
See 68a12a3.